src/mono/wasm/runtime/cwraps.ts

```diff
@@ -1,18 +1,18 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-import WasmEnableLegacyJsInterop from "consts:WasmEnableLegacyJsInterop";
-import {
+import type {
     MonoArray, MonoAssembly, MonoClass,
     MonoMethod, MonoObject, MonoString,
     MonoType, MonoObjectRef, MonoStringRef, JSMarshalerArguments
 } from "./types";
-import { Module } from "./imports";
-import { VoidPtr, CharPtrPtr, Int32Ptr, CharPtr, ManagedPointer } from "./types/emscripten";
+import type { VoidPtr, CharPtrPtr, Int32Ptr, CharPtr, ManagedPointer } from "./types/emscripten";
+import WasmEnableLegacyJsInterop from "consts:WasmEnableLegacyJsInterop";
+import { nativeLegacyJsInterop, Module } from "./imports";
```

Comment:
Why do we need to constants?

Reply:
so that in the future we could let the rollup trim flavor of the JS file.

Reply:
Sorry, I missed one letter in my question. What I'm asking, is why we need TWO constants/variables? Can this be united?